stopwatch.go

```diff
@@ -0,0 +1,147 @@
+package hrtime
+
+import (
+	"sync"
+	"sync/atomic"
+	"time"
+)
+
+// Span defines a time.Duration span
+type Span struct {
+	Start  time.Duration
+	Finish time.Duration
+}
```

Comment:
Ok, I understand. time.Duration is used as you keep time as time.Duration since 1970. I would recommend adding a note on that to func Now(). Maybe a slight note to Span comment as well.

Reply:
The reason `Now` doesn't define is that because the meaning of result depends on the OS. So the result from Windows and Linux are not comparable. This is the same reason it's using `time.Duration`, because it's relative some specific point in time... which may change even between restarts.

Reply:
I would recommend having a note about measuring time against some past moment either in here, in code, or as part of the README, where the general approach is explained.